algviz/picture/elements.py

```diff
@@ -0,0 +1,77 @@
+import abc
+from enum import Enum
+
+class ShapeException(TypeError):
+    def __init__(self, shape):
+        message = "{} not a recognized shape".format(str(shape))
+        super().__init__(message)
+
+class Shape(Enum):
+    #TODO: expand
+    CIRCLE = 0
+    RECT = 1
+
+    def draw(self, center, width, height, svg_engine, **kwargs):
+        if self is Shape.CIRCLE:
+            radius = min(width/2, height/2)
+            svg_engine.draw_circle(center, radius, **kwargs)
+        elif self is Shape.RECT:
```

Comment:
You're using an Enum instead of making two classes, probably so you can treat Enum attributes as static singletons.  Something is terribly wrong here.